### PR TITLE
all: prevent common large log lines

### PR DESCRIPTION
**Pull request by josharian**
- control/controlclient: do not periodically print full netmap
- ipn/ipnlocal: log number of packet filters rather than entire filter

### CODE REVIEW

**Comment by josharian** [comment]
I anticipate that this may be unpopular. I won't submit until I get an OK from both @bradfitz and @danderson.

**Comment by josharian** [comment]
Marking as draft, because @danderson points out that we need the u= part of the netmap line for grinder, at least in the short term.

**Comment by josharian** [comment]
Ready for review @danderson @bradfitz @crawshaw

**Review by danderson — APPROVED** [review]

**Pull request merged**
